juju/juju
# Fix LXD remote discovery for snap LXD.

This change fixes lxd cloud and credential discovery with snap lxd.
It also fixes credential discovery/matching for builtin lxd clouds.

## QA steps

Test localhost/lxd bootstrap
```
snap install lxd
juju bootstrap localhost # and lxd
```

Test bootstrapping remote cluster
```
lxc remote add mylxd 10.10.10.10
juju clouds # should show mylxd as a builtin lxd cloud
juju bootstrap mylxd
```

## Documentation changes

Updates required to lxd documentation.

## Bug reference

N/A
